examples/with-portals/pages/index.js

```diff
@@ -1,4 +1,3 @@
-import React from 'react'
 import Modal from '../components/Modal'
 
 export default () => <Modal />
```

Comment:
Please name this component.

Reply:
@Timer done. I was a bit hesitant to make more changes as it was my first PR. :)